pyCGM_Single/Pipelines.py

```diff
@@ -50,7 +47,21 @@ def butterFilter(data, cutoff, Fs):
     correction factor to the cuttoff frequency to compensate. Correction
     factor C = square root of 2 to the power of 1/n - 1, where n is equal to
     the number of passes. 
-    '''
+
+    Examples
+    --------
+    >>> data = [-1003.58361816, -1003.50396729, -1003.42358398, -1003.3425293,
+    ...         -1003.26068115, -1003.17810059, -1003.09484863, -1003.01080322,
+    ...         -1002.92608643, -1002.84063721, -1002.75445557, -1002.6675415,
+    ...         -1002.57995605, -1002.49157715, -1002.40252686, -1002.31274414]
+    >>> cutoff = 20
+    >>> Fs = 120
+    >>> butterFilter(data, cutoff, Fs)
+    array([-1003.58359202, -1003.50390817, -1003.42363665, -1003.34252947,
```

Comment:
not sure about this example, it seems like nothing changes.  It may be better to use numpy to create a function, add noise to is, and show that it filters that noise

Reply:
I changed the doctest to create a sin wave, add noise to it, and then filter it.